wave_benchmark/include/wave/benchmark/pose_measurement.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef WAVE_BENCHMARK_POSE_MEASUREMENT_HPP
+#define WAVE_BENCHMARK_POSE_MEASUREMENT_HPP
+
+#include <chrono>
+#include "wave/geometry/rotation.hpp"
+#include "wave/utils/math.hpp"
+
+namespace wave {
+
+using TimeType = std::chrono::steady_clock::time_point;
+
+struct Pose {
+    wave::Rotation rotation;
+    wave::Vec3 translation;
+};
+
+struct PoseMeasurement {
+    TimeType time_point;
+    int sensor_id;
+    Pose value;
+    PoseMeasurement(const TimeType &t, const int &s, const Pose &v)
+        : time_point{t}, sensor_id{s} {
+        value = v;
+    }
```

Comment:
Have you found a fix for not needing to put `value = v` in the constructor body? Ideally you want to avoid that if possible.